common/views/components/DateRange/DateRange.test.tsx

```diff
@@ -0,0 +1,75 @@
+import { mountWithTheme } from '@weco/common/test/fixtures/enzyme-helpers';
+import DateRange from './DateRange';
+
+describe('DateRange', () => {
+  it('renders a date + time range if the start/end are on the same day', () => {
+    const start = new Date('2022-09-18T12:00:00+0100');
+    const end   = new Date('2022-09-18T14:30:00+0100'); // prettier-ignore
```

Comment:
note: the `// prettier-ignore` comment means prettier doesn't remove the whitespace between `end =`, so the timestamps line up nicely.